.github/workflows/build_docker_images.yml

```diff
@@ -73,9 +73,9 @@ jobs:
 
       - name: Build and push CR3 extract diagram image
         if: steps.changes.outputs.cr3_extract == 'true'
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v4
         with:
           platforms: linux/amd64,linux/arm64
           context: atd-etl/cr3_extract_diagram
           push: true
-          tags: atddocker/atd-vz-cr3-extract:${{ github.ref == 'refs/heads/production' && 'production' || 'latest' }}
+          tags: atddocker/atd-vz-cr3-extract:${{ github.ref == 'refs/heads/master' && 'production' || 'latest' }}
```

Comment:
I had waffled on this back and forth heavily, before we had really started to get experience porting and adding DAGs. There's even a post it note on my monitor at home to ask y'all what you thought! I couldn't decide if we wanted to tie our images to release VZ Suite release events or if they should float to whenever we push changes on the main branch.

I am really glad you caught this. This is great, thank you!